.github/workflows/elementor-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Elementor Tests / Init
+
+on: [push, pull_request]
+
+jobs:
+  run:
+    runs-on: ubuntu-16.04
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@master
+      - name: Block non-permitted users
+        uses: StyleShit/action-whitelist-users@master
+        with:
+          whitelist-file: '.github/workflows/whitelist.txt'
+      - name: Elementor Tests / Init
+        env:
+          TOKEN: ${{ secrets.DEVOPS_TOKEN }} # GitHub access token to access the remote repo.
+          REMOTE_TESTS_REPO: ${{ secrets.REMOTE_TESTS_REPO }} # Remote tests repo, e.g. `owner/repo-name`.
+          HEAD_REPO: ${{ github.event.pull_request.head.repo.full_name || github.repository }} # The HEAD repo that initiated the push/PR.
```

Comment:
if the `github.event.pull_request.head.repo.full_name` is not defined we don't need to run the workflow

Reply:
That's empty only if it's a push and not a PR...

Reply:
so you don't need to check if the event type is `pull_request `?

Reply:
I don't need to because I know that if the value is null, it has to be a push event.